client/me/purchases/cancel-private-registration/index.jsx

```diff
@@ -20,7 +20,8 @@ import SimpleNotice from 'notices/simple-notice';
 const CancelPrivateRegistration = React.createClass( {
 	getInitialState() {
 		return {
-			disabled: false
+			disabled: false,
+			cancelling: false
```

Comment:
I think we do. For this case the output will be the same regardless, but I think it is important to be a bit more explicit about the attributes. Afaik we have been explicit about our attributes and I'm following the convention. We can just use the "cancelling" attribute for this case, but I think being explicit is more of an example. This is open to discussion though.